postcss.config.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  plugins: {
+    tailwindcss: {},
+    autoprefixer: {},
```

Comment:
I guess you just copied this config from an interesting article and that's why we're having `autoprefixer` and `postcss`.

Reply:
deleted

Reply:
deleted